libraries/c_sdk/aws/defender/test/unit/aws_iot_tests_defender_unit.c

```diff
@@ -55,7 +55,7 @@
  * @brief Default Invalid Metrics Group.
  * Used by the SetMetrics_with_invalid_metrics_group unit test.
  */
-#define AWS_IOT_DEFENDER_DEFAULT_INVALID_METRICS_GROUP    ( 10000 )
+#define AWS_IOT_DEFENDER_DEFAULT_INVALID_METRICS_GROUP    ( 10 )
```

Comment:
Resolves the following `-Woverflow` warning:
```
/home/ubuntu/Repos/amazon-freertos/libraries/c_sdk/aws/defender/test/unit/aws_iot_tests_defender_unit.c: In function 'TEST_Defender_Unit_SetMetrics_with_invalid_metrics_group_':
/home/ubuntu/Repos/amazon-freertos/libraries/c_sdk/aws/defender/test/unit/aws_iot_tests_defender_unit.c:58:59: warning: unsigned conversion from 'int' to 'AwsIotDefenderMetricsGroup_t' {aka 'enum <anonymous>'} changes value from '10000' to '16' [-Woverflow]
   58 | #define AWS_IOT_DEFENDER_DEFAULT_INVALID_METRICS_GROUP    ( 10000 )
      |                                                           ^~~~~~~~~
/home/ubuntu/Repos/amazon-freertos/libraries/c_sdk/aws/defender/test/unit/aws_iot_tests_defender_unit.c:193:62: note: in expansion of macro 'AWS_IOT_DEFENDER_DEFAULT_INVALID_METRICS_GROUP'
  193 |     AwsIotDefenderError_t error = AwsIotDefender_SetMetrics( AWS_IOT_DEFENDER_DEFAULT_INVALID_METRICS_GROUP,
      |                                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
I see. I was going to ask what this is for :)